components/tabs/demo/tabs.html

```diff
@@ -69,56 +55,19 @@ <h2>Tabs (new structure)</h2>
+			<h2>Tabs (with custom tab)</h2>
 
-			<div style="margin-bottom: 30px;">
-				<d2l-button-subtle id="add-old" text="Add"></d2l-button-subtle>
-				<d2l-button-subtle id="add-selected-old" text="Add Selected"></d2l-button-subtle>
-				<d2l-button-subtle id="remove-old" text="Remove"></d2l-button-subtle>
-				<d2l-button-subtle id="remove-multiple-old" text="Remove Multiple"></d2l-button-subtle>
-			</div>
-			<script>
-				let newPanelIdOld = 0;
-				const addPanelOld = (selected, tabs) => {
-					newPanelIdOld += 1;
-					const panel = document.createElement('d2l-tab-panel');
-					panel.selected = selected;
-					panel.text = `New Panel ${newPanelIdOld}`;
-					panel.textContent = `Content for new panel ${newPanelIdOld}`;
-					const panels = [...tabs.querySelectorAll('d2l-tab-panel')];
-					if (panels.length < 2) tabs.appendChild(panel);
-					else tabs.insertBefore(panel, panels[1]);
-				};
-				const removePanelOld = (tabs) => {
-					const panels = [...tabs.querySelectorAll('d2l-tab-panel')];
-					if (panels.length === 0) return;
-					if (panels.length === 1) tabs.removeChild(panels[0]);
-					else tabs.removeChild(panels[1]);
-				};
-				const removePanelsOld = (tabs) => {
-					const panels = [...tabs.querySelectorAll('d2l-tab-panel')];
-					if (panels.length === 0) return;
-					if (panels.length === 1) tabs.removeChild(panels[0]);
-					else {
-						tabs.removeChild(panels[1]);
-						tabs.removeChild(panels[0]);
-					}
-				};
-
-				document.querySelector('#add-old').addEventListener('click', () => addPanelOld(false, document.querySelector('#withSlotOld').querySelector('d2l-tabs')));
-				document.querySelector('#add-selected-old').addEventListener('click', () => addPanelOld(true, document.querySelector('#withSlotOld').querySelector('d2l-tabs')));
-				document.querySelector('#remove-old').addEventListener('click', () => removePanelOld(document.querySelector('#withSlotOld').querySelector('d2l-tabs')));
-				document.querySelector('#remove-multiple-old').addEventListener('click', () => removePanelsOld(document.querySelector('#withSlotOld').querySelector('d2l-tabs')));
-			</script>
-
-			<d2l-demo-snippet id="withSlotOld">
+			<d2l-demo-snippet>
 				<template>
-					<d2l-tabs>
-						<d2l-tab-panel text="Biology">Tab content for Biology</d2l-tab-panel>
-						<d2l-tab-panel text="Chemistry">Tab content for Chemistry</d2l-tab-panel>
-						<d2l-tab-panel text="Earth Sciences">Tab content for Earth Sciences</d2l-tab-panel>
-						<d2l-tab-panel text="Physics">Tab content for Physics</d2l-tab-panel>
-						<d2l-tab-panel text="Math">Tab content for Math</d2l-tab-panel>
+					<d2l-tabs text="Courses">
+						<d2l-tab-custom id="all" slot="tabs"><div style="color: purple;">All</div></d2l-tab-custom>
+						<d2l-tab-custom id="biology" slot="tabs" selected>
+							Biology <d2l-count-badge has-tooltip number="100" size="small" tab-stop text="100 new notifications" type="notification"></d2l-count-badge>
```

Comment:
Re `tab-top`...  [same as discussion in the other PR](https://github.com/BrightspaceUI/core/pull/5467#discussion_r2031414958), we should define expectations/requirements re focusables in custom tab handles.